docs/mdbook/default.nix

```
@@ -369,6 +370,21 @@ pkgs.stdenv.mkDerivation (finalAttrs: {
 
   passthru = {
     copy-docs = pkgs.writeShellScript "copy-docs" docs.commands;
+    readme =
+      runCommand "readme"
+        {
+          start = "<!-- START DOCS -->";
+          end = "<!-- STOP DOCS -->";
+          baseurl = "https://nix-community.github.io/nixvim/";
+          src = ../../README.md;
+        }
+        ''
+          # extract relevant section of the README
+          sed -n "/$start/,/$end/p" $src > $out
+          # replace absolute links
+          # TODO: replace .html with .md
+          substituteInPlace $out --replace "$baseurl" "./"
```

```suggestion
          substituteInPlace $out --replace-quiet "$baseurl" "./"
```

If you want it to be silent